### PR TITLE
Add documentation for OASA Telematics greek public transport sensor c…

### PR DESCRIPTION
**Pull request by panosmz**
**Description:**
Documentation for the OASA Telematics greek public transport sensor component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** <https://github.com/home-assistant/home-assistant/pull/21697>

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.oasa_telematics.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+description: "Instructions on how to integrate bus and trolley arrival data for Greek OASA Telematics within Home Assistant."
+date: 2019-03-05 21:52
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: oasa.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.89
```
Comment (klaasnicolaas) [inline source/_components/sensor.oasa_telematics.markdown]:
```suggestion
ha_release: 0.90
```

File: source/_components/sensor.oasa_telematics.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+layout: page
+title: "OASA Telematics"
+description: "Instructions on how to integrate bus and trolley arrival data for Greek OASA Telematics within Home Assistant."
+date: 2019-03-05 21:52
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: oasa.png
+ha_category: Transport
```
Comment (klaasnicolaas) [inline source/_components/sensor.oasa_telematics.markdown]:
```suggestion
ha_category:
  - Transport
  - Sensor
```

File: source/_components/sensor.oasa_telematics.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<http://telematics.oasa.gr/api/?act=webGetRoutes&p1=LINE_CODE>
+(Replace "LINE_CODE" with the "LineCode" you copied from the first link) find the route you need and copy the `RouteCode` field.
+
+Next, get the `stop_id` from this link: 
+
+<http://telematics.oasa.gr/api/?act=webGetStops&p1=ROUTE_CODE>
+(Replace "ROUTE_CODE" with the "RouteCode" you got from the previous link) find the stop you need and copy the `StopID` field. The route must pass from this stop in order for the sensor to work.
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_components/sensor.oasa_telematics.markdown:46]: Pherhaps adding the `name` variable

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/sensor.oasa_telematics.markdown (hunk 3 above)
Comment (panosmz) [reply source/_components/sensor.oasa_telematics.markdown:46]: added `name` variable. thanks for the feedback

**Pull request closed without merging**